_projects/mohammad_project.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Improved Assessment of Brain dynamic Functional Connectivity
+status: active
+layout: project
+people:
+  - Mohammad
```

Comment:
This should match your [name in the people.yml file](https://github.com/neurodatascience/neurodatascience.github.io/blob/gh-pages/_data/people.yml#L120). I think it does!

Reply:
I see. Yes it does!